### ISSUE TITLE
Push notifications

### ISSUE THREAD

**Issue by mastercoms** [mastercoms/mastercomfig#174]
Add push notifications to the download website, so that we can push new releases through cURL, notifying people through their browser that there has been a new update.

**Comment by micwoj92** [comment]
I know it's probably not what you meant, but maybe recommend users to subscribe to releases rss feed?
https://github.com/mastercomfig/mastercomfig/releases.atom

I use it for few projects and it works fine.

**Comment by mastercoms** [comment]
Yeah, that could be a solution for now, but I meant something like service workers.
